pylint_odoo/test_repo/broken_module2/tests/data/odoo_data_noupdate_1.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+    </data>
+    <data>
+    </data>
+</odoo>
```

Comment:
@dreispt 
Yes, the message only show when inside the `<odoo>` tag contains **_only one_** `<data>` tag

The detected cases are

```xml
<odoo>
  <data noupdate="0">
    ...
  </data>
</odoo>
```
```xml
<odoo>
  <data noupdate="1">
    ...
  </data>
</odoo>
```
```xml
<odoo>
  <data>
    ...
  </data>
</odoo>
```